src/persist-client/src/lib.rs

```diff
@@ -387,6 +374,50 @@ impl ConsensusKnobs for PersistConfig {
     }
 }
 
+/// Persist configurations backed by LaunchDarkly.
```

Comment:
supernit: might be nice to separate out the idea of dynamic config from the LD implementation details

Reply:
done!